tests/installation/partitioning_togglehome.pm

```diff
@@ -20,28 +20,28 @@ use version_utils qw(is_storage_ng is_leap is_sle);
 sub run {
     record_soft_failure 'boo#1093372' if (!get_var('TOGGLEHOME') && is_leap('15.1+'));
     send_key $cmd{guidedsetup};
-    assert_screen [qw(partition-scheme  inst-partition-radio-buttons)];
+    assert_screen [qw(partition-scheme inst-partition-radio-buttons)];
     if (is_storage_ng) {
         assert_screen 'partition-scheme';
         send_key $cmd{next};
         installation_user_settings::await_password_check if get_var('ENCRYPT');
     }
     # For s390x there was no offering of separated home partition until SLE 15 See bsc#1072869
-    if (!check_var('ARCH', 's390x') or is_storage_ng()) {
-        if (!check_screen 'disabledhome', 0) {
-            # toggle_home hotkey can change
-            if (check_screen('inst-partition-radio-buttons', 0)) {
-                $cmd{toggle_home} = 'alt-r';
-            }
-            elsif (check_screen('proposed-separated-swap', 0)) {
-                $cmd{toggle_home} = 'alt-o';
-            }
-            send_key $cmd{toggle_home};
+    if ((!check_var('ARCH', 's390x') or is_storage_ng()) and !match_has_tag('disabledhome')) {
+        my @tags = qw(enabledhome-partition enabledhome-volume);
```

Comment:
am I missing something or this variable used only once in line 37 ? 
In this case it will be more clear to just use string inline. There is no benefits from this variable ( no code reuse , also name "tags" is not bring any additional info ). 
With same luck you can declare two variables for line 30 : `my $arch='s390x'` and `my $matched_tag='disabledhome'`

Reply:
right, this was from a previous approach when I was conditionally appending a third tag later. Updated.

I did not understand your second comment regarding L30 though.